rdtools/__init__.py

```diff
@@ -14,15 +14,15 @@
 from rdtools.filtering import tcell_filter
 from rdtools.filtering import clip_filter
 from rdtools.filtering import normalized_filter
-from rdtools.soiling import soiling_srr
-from rdtools.soiling import monthly_soiling_rates
-from rdtools.soiling import annual_soiling_ratios
+# from rdtools.soiling import soiling_srr
+# from rdtools.soiling import monthly_soiling_rates
+# from rdtools.soiling import annual_soiling_ratios
 from rdtools.plotting import degradation_summary_plots
 from rdtools.plotting import soiling_monte_carlo_plot
 from rdtools.plotting import soiling_interval_plot
 from rdtools.plotting import soiling_rate_histogram
 from rdtools.plotting import availability_summary_plots
```

Comment:
Should we comment out the soiling/availability plotting function imports too?  Do they need a warning?

Reply:
I'm leading toward commenting them out of `__init__.py`, adding a warning at the function level, and adding a note to the function docstrings.

Reply:
Done, example: https://rdtools--213.org.readthedocs.build/en/213/generated/rdtools.plotting.soiling_interval_plot.html